### PR TITLE
Feat/28 did qrcode

### LINKED ISSUE #28: Allow scanning of DID to get endpoint for issuing and presentations
Instead of scanning the endpoint and pressing Allow/Deny for:

> Do you trust this host?

scan a DID where the endpoint is included in the DID document as a service with the DID verifiable.

### PR DESCRIPTION
**Pull request by sgreenbury**
Closes #28.

### CODE REVIEW

File: lib/app/pages/qr_code/bloc/qrcode.dart [hunk 1]
```diff
@@ -63,15 +68,47 @@ class QRCodeBloc extends Bloc<QRCodeEvent, QRCodeState> {
     late final uri;
 
     try {
-      uri = Uri.parse(event.data);
+      // Decode JSON string
+      final didCode = jsonDecode(event.data);
+      final did = didCode['did'];
+      final route = didCode['route'];
+      final uuid = didCode['uuid'];
```
Comment (sgreenbury) [inline lib/app/pages/qr_code/bloc/qrcode.dart:75]: Check whether `id` now

Comment (sgreenbury) [reply lib/app/pages/qr_code/bloc/qrcode.dart]: Now updated.

**Review by edchapman88 — COMMENTED** [review]
looks good!

**Review by edchapman88 — APPROVED** [review]
looks good!

**Pull request merged**